src/main/kotlin/com/openlattice/shuttle/pods/ShuttleStreamSerializersPod.kt

```diff
@@ -0,0 +1,17 @@
+package com.openlattice.shuttle.pods
```

Comment:
pretty sure this can all go away if we move `Integration` into `Conductor-client`. I know this involves moving `Flight`, `Integration`, and `FlightPlanParameters` as well as `EntityDefinition`, `AssociationDefinition` and possibly one or two more classes but since they're all effectively becoming Hazelcast objects and `shuttle` is already coupled to `conductor-client` with these changes, shouldn't they be in `conductor-client`?